eng/doc/fips/README.md

```diff
@@ -358,11 +358,13 @@ The OpenSSL version present when building a program does not have to match the O
 
 This feature does not require any additional configuration, but it only works with OpenSSL versions known and supported by the Go toolchain.
 
-### TLS with FIPS-approved settings
+### TLS with FIPS-compliant settings
 
-The Go TLS stack will automatically use OpenSSL crypto primitives when running in FIPS mode. Yet, the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
+The Go TLS stack will automatically use crypto primitives from the selected crypto backend. Yet, this isn't enough for FIPS compliance: the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
```

Comment:
Sorry, one more thing in the doc. 😄 I think the `compatibility with old devices` note should be moved below this paragraph to make it 100% clear that the FIPS restrictions themselves cause the compatibility issue, not our approach to it. (And definitely not just the `import` approach!)

Reply:
Good point. Thanks!